src/components/Button/Button.tsx

```diff
@@ -1,18 +1,16 @@
 import * as React from 'react';
 import * as style from './Button.scss';
 
-interface IOwnProps {
-  handleClick: () => void;
+export interface IButtonProps {
```

Comment:
Why export?

Reply:
So it can be referenced elsewhere and extended